client/src/components/Button/SurveyButton.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from "prop-types";
+import styled from "styled-components/macro";
+
+const sizes = {
+  mini: {
+    padding: "4.5vw",
+  },
+  small: {
+    padding: "5vw",
+  },
+  medium: {
+    padding: "5.5vw",
+  },
+  large: {
+    padding: "6vw",
+  },
```

Comment:
If possible, it would be great to have this relative to the parent to allow adding this component to different viewports.

Reply:
We created an issue for this and will work on that later :)